lookout/style/format/feature_extractor.py

```diff
@@ -484,17 +482,16 @@ def _classify_vnodes(self, nodes: Iterable[VirtualNode], path: str) -> Iterable[
             line_offset = 0
             for i, line in enumerate(lines[:-1]):
                 # `line` contains trailing whitespaces, we add it to the newline node
-                newline = line + sep
                 start_offset = node.start.offset + line_offset
                 start_col = node.start.col if i == 0 else 1
                 lineno = node.start.line + i
                 yield VirtualNode(
-                    newline,
+                    line,
                     Position(start_offset, lineno, start_col),
-                    Position(start_offset + len(newline), lineno, start_col + len(newline)),
+                    Position(start_offset + len(line), lineno + 1, 1),
```

Comment:
@vmarkovtsev and @m09 please pay attention here.
I think it was a bug because it behaved differently in comparison with pure `_parse_file()` call: token with a new line ends at the beginning of next line. 

If it was on purpose we should change it in `_parse_file()` method. 

I see that this change can affect Hugo's visualizer, some of the features like `_FeatureDiffLine`.